Cargo.toml

```diff
@@ -1,10 +1,13 @@
 [package]
-description = "Parity Ethereum client"
-name = "parity-ethereum"
+description = "OpenEthereum"
+name = "openethereum"
 # NOTE Make sure to update util/version/Cargo.toml as well
-version = "2.5.13"
+version = "3.0.1"
```

Comment:
3.1 ?

Reply:
Not sure what is practice, do we update to 3.1.0 just before release or we are putting the next version after release. I would put 3.1 as last commit before our release, but for version, I don't have a strong opinion.